pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -144,3 +148,30 @@ func removeOldLogs() error {
 	}
 	return nil
 }
+
+func pullBundleImage(preset preset.Preset) error {
+	imgHandler := image.OCIImageHandler{
+		ImageName: image.GetImageName(preset),
+		ImageTag:  version.GetBundleVersion(),
```

Comment:
we can modify `GetBundleVersion` to also take a preset and return the podman version in case of podman preset
```
DEBU Accessing "quay.io/crcont/podman-vfkit-bundle:4.10.14" failed: reading manifest 4.10.14 in quay.io/crcont/podman-vfkit-bundle: manifest unknown: manifest unknown
initializing source docker://quay.io/crcont/podman-vfkit-bundle:4.10.14: reading manifest 4.10.14 in quay.io/crcont/podman-vfkit-bundle: manifest unknown: manifest unknown
```

Reply:
Pushing both bundle version and crc version as tags for the same image is quite confusing, there is no hint why we have `podman-vfkit-bundle:2.5.0` being newer than `podman-vfkit-bundle:4.0.2`. Might be an indication that crc version -> bundle version should not be handled as tags on the image.
Also, what happens the day we release `crc` version `4.0.2`?